Flyte Scribe/presence.ts

```diff
@@ -0,0 +1,30 @@
+var presence = new Presence({
+    clientId: "707632555612045413"
+  }),
+  presenceData: presenceData = {
+    largeImageKey: "logo"
+  },
+  customData = false;
+
+presence.on("UpdateData", async () => {
+  customData = false;
```

Comment:
custromData is never set to true, so why is it needed?

Reply:
During development, edo/flyte had a page which had some custom data in it depending on the day.
Shortly before the pull request was made, edo had decided it was not needed there.
I somehow forgot to remove that bit of line, and fixed other lines that were dependent on this variable.
It's now fixed in https://github.com/PreMiD/Presences/pull/1511/commits/461c43ae697bbbbe89b9dadc4990f93c039e5b50.